src/legacy/server/saved_objects/routes/bulk_create.ts

```diff
@@ -17,9 +17,33 @@
  * under the License.
  */
 
+import Hapi from 'hapi';
 import Joi from 'joi';
+import { SavedObjectsClient } from '../';
+import { Prerequisites, SavedObjectReference } from './types';
 
-export const createBulkCreateRoute = prereqs => ({
+interface SavedObject {
+  type: string;
+  id?: string;
+  attributes: { [key: string]: any };
+  version?: string;
+  migrationVersion?: { [key: string]: string };
+  references: SavedObjectReference[];
+}
+
+// @ts-ignore - Hapi.Request "query" doesn't allow non string types
```

Comment:
We can use something like `Pick<T, Exclude<keyof T, 'query'>>;`. We don't mute typescript errors at least. Is there an issue created in `@types/hapi` for this problem?